### PR TITLE
.NET Core 2.2 updates

### PR DESCRIPTION
**Pull request by ahardin-rh**

### CODE REVIEW

**Comment by omajid** [comment]
cc @aslicerh @tmds

**Review by aslicerh — COMMENTED** [review]
Line 62: The image list also needs to be updated.

There's also been some changes to the environment variables. Are you going to make those changes, or should one of us?

File: using_images/s2i_images/dot_net_core.adoc [hunk 1]
```diff
@@ -33,17 +33,17 @@ and Python from within {product-title}.
...
 * .NET Core version 1.1
 * .NET Core version 1.0
 * Supported on Red Hat Enterprise Linux (RHEL) 7
 ifdef::openshift-enterprise[]
 and {product-title} versions 3.3 and later
 endif::openshift-enterprise[]
 
-For release details related to .NET Core version 2.1, see
-link:https://access.redhat.com/documentation/en-us/net_core/2.1/html/release_notes_for_containers/[Release
+For release details related to .NET Core version 2.2, see
```
Comment (omajid) [inline using_images/s2i_images/dot_net_core.adoc]: We currently have release notes for 2.2 and 1.1 and 1.0. Only version 2.1 doesn't have a link to release notes any more. Is that intentional? Should we keep a link to the 2.1 release notes?

Comment (tobydrake7) [reply using_images/s2i_images/dot_net_core.adoc]: Yes, I may have been confusing in my notes for changes to make. Until 2.1 is EOL'd it is probably best to keep the 2.1 release notes available.

Comment (tobydrake7) [reply using_images/s2i_images/dot_net_core.adoc]: Also, Ashley, we will have one or two new environment variables to add to the table. I will supply this new information to you.

**Comment by tmds** [comment]
@ahardin-rh is there some OpenShift documentation that deals with configuring authentication for the registry.redhat.io server?

**Comment by ahardin-rh** [comment]
@tmds Yes, we have this topic: https://docs.openshift.com/container-platform/3.11/install_config/configuring_red_hat_registry.html

**Review by tmds — APPROVED** [review]
Lgtm, thank you!

**Review by omajid — APPROVED** [review]
LGTM! Thanks

**Comment by ahardin-rh** [comment]
Thanks, all! I confirmed with @tobydrake7 that this is ready to publish.

**Pull request merged**